### PR TITLE
Add etcd-io org

### PR DESCRIPTION
**Pull request by cblecker**
This adds the etcd-io org under administration

### CODE REVIEW

**Comment by cblecker** [comment]
/hold
the github bot needs to join the org first

File: config/etcd-io/org.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      raft: triage
+      website: triage
+  reviewers-etcd:
+    description: ""
+    members:
+    - fuweid
+    - jmhbnz
+    privacy: closed
+    repos:
+      etcd: triage
```
Comment (jmhbnz) [inline config/etcd-io/org.yaml:138]: Need to double check these `reviewers-etcd` permissions reflect a recent update https://github.com/etcd-io/website/issues/730.

Comment (serathius) [reply config/etcd-io/org.yaml:138]: +1

File: config/restrictions.yaml [hunk 2]
```diff
@@ -364,4 +364,15 @@ restrictions:
...
+    allowedRepos:
+    - "^bbolt$"
+    - "^dbtester$"
+    - "^discovery.etcd.io$"
+    - "^discoveryserver$"
+    - "^etcd$"
+    - "^gofail$"
+    - "^jetcd$"
+    - "^raft$"
+    - "^website$"
```
Comment (jmhbnz) [inline config/restrictions.yaml:377]: This list appears to be missing the following three repositories which we recently added `OWNERS` files to.

- [ ] `etcd-io/protodoc` 
- [ ] `etcd-io/etcdlabs` 
- [ ] `etcd-io/govanityurls`

**Comment by wenjiaswe** [comment]
@cblecker Just a friendly ping. Really appreciate your help working on this, or please let us know if it's easier for us to work on this. Thanks!

**Comment by cblecker** [comment]
@wenjiaswe Sorry for the delay on this. We're actually working through some technical difficulties getting the bot added to the org, which is what is holding this up. We're in contact with GitHub and will let you know once we figure out a path forward. Thanks for your patience!

**Comment by wenjiaswe** [comment]
Sure, thank you very much, just want to make sure there is nothing pending on us.

**Comment by cblecker** [comment]
@wenjiaswe An update: we are still working on this. We made another pre-requisite change today, moving this org out of the CNCF enterprise account into the Kubernetes one. Still working with GH on some of the other pieces.

**Comment by sftim** [comment]
Not sure if we should mark this as SIG etcd?

**Comment by wenjiaswe** [comment]
> Not sure if we should mark this as SIG etcd?

@sftim sure, that's a good idea just to make sure it's under SIG-etcd's radar. 

I just want to make sure, marking this as SIG-etcd won't take this off the attention list of github admins.

**Comment by wenjiaswe** [comment]
> Wenjia An update: we are still working on this. We made another pre-requisite change today, moving this org out of the CNCF enterprise account into the Kubernetes one. Still working with GH on some of the other pieces.

@cblecker Thank you for the update, and thank you very much for helping us to set this up!

**Comment by cblecker** [comment]
@jmhbnz @wenjiaswe Sorry for the delay on this one. Onboarding a new org isn't something we do frequently, and it's exposed some issues with the process. We're still waiting to hear back from out contacts at GH on a blocking issue (I can't get into specifics publicly as there are some security implications), but we haven't forgot about this one at all. Very much hope to have this resolved soon as folks are getting back from the winter break. Really appreciate your patience!

**Comment by cblecker** [comment]
@jmhbnz @wenjiaswe I have good news! The issues that were preventing this change from moving forward have been resolved!

I've taken a fresh dump of the current state, and this is ready for review. :)

**Review by jmhbnz — APPROVED** [review]
/lgtm

Many thanks for your help on this @cblecker 🙏🏻

**Comment by cblecker** [comment]
/hold cancel 

Let's go!

**Pull request merged**